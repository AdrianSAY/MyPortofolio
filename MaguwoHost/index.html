<!DOCTYPE html>

<html lang="en">

    <head>
         <meta charset="UTF-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>MaguwoHost </title>
         <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;400;700&display=swap" rel="stylesheet">
         <link rel="stylesheet" href="css/styles.css">
  
    </head>
          
    <body>
         <header>
             <section class="header-navbar">


                 <section class="logo-home">
                         <img src="images/logosite.png" alt="">
                         <h1>MaguwoHost</h1>             
                     <nav>
                        <ul>
                         <li><a href="#">Home</a></li>
                         <li><a href="#">About</a></li>
                         <li><a href="#">Pricing</a></li>
                         <li><a href="#">Domain</a></li>
                         <li><a href="#">Hosting</a></li>

                         </ul>
                     </nav>
                </section>

                <section class="contact">
                      <p>Do you need help? Just Email,Live chat or Call us</p>
                      <div class="live-chat">
                      <div class="live-chat1">Live Chat</div>

                       <div class="tel">6284-6534</div>
                      </div>            
                    </section>
             </section>

             <section class="hero">
                 <img src="images/hero.png" alt="">

                 <div class="hero-content">
                    <div class="container">
                     <h2>Just a Proffessional Web Hosting Services</h2>
                     <p class="paragraph-hero">moder - fast -reliable</p>
                     <img  src="images/logohero.png" alt="">
                    </div>
                    <div class="check-button">
                        <div class="button-personalize"></div>
                              <input class="inputemail" type="email" name="Your Domein Name..." placeholder="Your Domein Name...">
                              <button class="buttoncheck" type="submit">Check Availability</button>
                    
                        </div>
                    </div>
                 </div>
             </section>

             <section class="services">
                  <section class="webhost">
                      <img class="prrr" src="images/webhost services.png" alt="">
                       <div class="webhost-services">
                           <img src="images/logowebservices.png" alt="">
                           <h3>#1 Webhost Services</h3>
                           <p>This is Photoshop's version of Lorem Ipsum.
                               Proin gravida nibh ve. Velit auctor aliquet.
                               Aenean sollicitudin,larem quis bibendum auctor,
                               nisi elit consequat ipsum,nec sagittis sem nibh id elit.
                           </p>
                           <button>More</button>
                       </div>
                    </section>
                    <section class="domain">
                        <img class="webhost">
                        <img class="prrr" src="images/domain.png" alt="">
                         <div class="domain-services">
                             <img src="images/logodomain.png" alt="">
                             <h3>#1 Domain Services</h3>
                             <p>This is Photoshop's version of Lorem Ipsum.
                                Proin gravida nibh ve. Velit auctor aliquet.
                                Aenean sollicitudin,larem quis bibendum auctor,
                                nisi elit consequat ipsum,nec sagittis sem nibh id elit.</p>
                                <button>More</button>
                         </div>
                    </section>   

                  
             </section>

             <section class="ourfeautures">
                <div class="title-subtitle">
                <h3 class="ourfeautures-title">Our Features</h3>
                <h4 class="ourfeautures-subtitle">Our Best Features</h4>
            </div>
                <img src="images/ourfeature background.png" alt="">

                      <din class="ourfeatures-container">
                         <div class="livesuport">
                             <img src="images/live suport.png" alt="">
                             <hr>
                             <h3 class="yellow">Live Support</h3>
                             <p class="item">This is Photoshop's version of Lorem Ipsum.
                                 Proin gravida nibh vel velit auctor aliquet.
                             </p>
                         </div>
                           
                         <div class="cloudtehnology">
                              <img src="images/cloud.png" alt="">
                              <hr>
                              <h3 class="yellow">Cloud Technology</h3>
                              <p class="item">This is Photoshop's version of Lorem Ipsum.
                                Proin gravida nibh vel velit auctor aliquet.
                            </p>

                         </div>

                         <div class="hitechdatabase">
                              <img src="images/hi tech data base.png" alt="">
                              <hr>
                              <h3 class="yellow">Hi Tech Data Base</h3>
                              <p class="item">This is Photoshop's version of Lorem Ipsum.
                                Proin gravida nibh vel velit auctor aliquet.
                            </p>
                         </div>

                         <div class="livemonitoring">
                             <img src="images/live monitoring.png" alt="">
                             <hr>
                             <h3 class="yellow">Hi Tech Data Base</h3>
                              <p class="item">This is Photoshop's version of Lorem Ipsum.
                                Proin gravida nibh vel velit auctor aliquet.
                            </p>

                         </div>
                      </din>
             </section>

             <section class="ourclients">
                 <section class="ourclients-content">
                     <div class="ourclients-text">
                         <h4 class="ourclients">Our Clients</h4>
                         <p class="ourclients-subtitle">This is Photoshop's version of Lorem Ipsum.
                            Proin gravida nibh vel velit auctor aliquet.</p>
                     </div>
                     <div class="ourclients-images">
                         <img class="imagine" src="images/client1.png" alt="">
                         <img class="imagine" src="images/client2.png" alt="">
                         <img class="imagine" src="images/client3.png" alt="">
                         <img class="imagine" src="images/client4.png" alt="">
                     </div>
                 </section>
             </section>

             <section class="review">
                 <div class="clients-review">
                     <div class="poza">
                          <img src="images/reviewbackground.png" alt="">
                     </div>
                     
                     <div class="poza1">
                        <img src="images/reviewbackground.png" alt="">
                   </div>
                   <div class="poza1">
                    <img src="images/reviewbackground.png" alt="">
               </div>
                 </div>
             </section>

             <section class="plan-pricing">
                 <img class="plan-pricing-img" src="images/fffffff.png" alt="">
                       
                        <div class="plan-pricing-text">
                            <h5>Do you want to view Plans & Pricing?</h5>
                            <p class="plan-pricing-subs">Follow the button in the right section</p>

                        </div>
                        <div class="buttonfff">
                            <button>Plans&Pricing</button></div>
                             
                 
             </section>
    
            <footer>
                <div class="footerimg">
                <img src="images/footer.png" alt="">
            </div>
            </footer>
            
         </header>
    
    </body> 
            
    
 
</html>