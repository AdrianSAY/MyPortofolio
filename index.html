<!DOCTYPE html>

<html lang="en">

    <head>
         <meta charset="UTF-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <title>SERBAN ADRIAN</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;700&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap" rel="stylesheet">
         <link rel="stylesheet" href="css/styles.css">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" crossorigin="anonymous">
    </head>
          
    <body>
         <section class="header">
               <nav class="navbar container  navbar-expand-lg navbar-light ">
                        
                              <a class="navbar-brand" href="#"><img src="images/logome.png" style="border-radius: 30%;" alt="" width="60px;"></a>
                              <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
                              <span class="navbar-toggler-icon"></span>
                              </button>
                         
                         <div class="collapse navbar-collapse" id="navbarNav">
                              <ul class="navbar-nav">
                                   <li class="nav-item active">
                                        <a class="nav-link" href="index.html#work">WORK <span class="sr-only">(current)</span></a>
                                   </li>
                                   <li class="nav-item">
                                        <a class="nav-link" href="index.html#aboutme">ABOUT ME</a>
                                   </li>
                                   <li class="nav-item">
                                        <a class="nav-link" href="index.html#myprofile">PROFILE</a>
                                   </li>
                              </ul>
                         </div>
               </nav>
         </section>
         <section class="hero">
          
              <div class="title container">
                   <h2>Hello, I'm</h2>
                   <h1>Serban Adrian</h1>
                   <p>Jr.Front-End Developer</p>
                   <div class="hero-social ">
                        <a href="https://www.facebook.com/serban.adrian.923"><img src="images/facebook.png" alt=""></a>
                        <a href="https://www.instagram.com/adrianserban91/"><img src="images/instagram.png" alt=""></a>
                        <a href="https://www.linkedin.com/in/adrian-serban-89870722a"><img src="images/linkedin.png" alt=""></a>
                   </div>
               </div>
         </section>
         <section class="work container" id="work">
          <h2>
               <span>
                   My Work
                   <hr>
               </span>
           </h2>
              <div class="row">
                    <a href="Modest/index.html" style="text-decoration: none;" class="col-sm-12 col-md-6 col-lg-4"><div class="overlay-section">
                         <img src="images/Modest .png" alt="Avatar" class="image">
                         <div class="overlay">MoDEST #6</div>
                    </div></a>
                    

                    <a href="Golden/index.html" style="text-decoration: none;" class="col-sm-12 col-md-6 col-lg-4"><div class="overlay-section">
                         <img src="images/Golden-One-Page-Web-Template.jpg" alt="Avatar" class="image">
                         <div class="overlay">GOLDEN #5</div>
                    </div></a>

                    <a href="Mogo/index.html" style="text-decoration: none;" class="col-sm-12 col-md-6 col-lg-4"><div class="overlay-section">
                         <img src="images/mogo-free-website-template-full-image-580x3021.jpg" alt="Avatar" class="image">
                         <div class="overlay">MOGO #4</div>
                    </div></a>

                    <a href="MaguwoHost/index.html" style="text-decoration: none;" class="col-sm-12 col-md-6 col-lg-4"><div class="overlay-section">
                         <img src="images/MaguwoHostTemplateImage.png" alt="Avatar" class="image">
                         <div class="overlay">MaguwoHOST #3</div>
                    </div></a>

                    <a href="Reen/index.html" style="text-decoration: none;" class="col-sm-12 col-md-6 col-lg-4"><div class="overlay-section">
                         <img src="images/ReenImageTemplate.png" alt="Avatar" class="image">
                         <div class="overlay">REEN #2</div>
                    </div></a>

                    <a href="HomeTech/index.html" style="text-decoration: none;" class="col-sm-12 col-md-6 col-lg-4"><div class="overlay-section">
                         <img src="images/home tech.png" alt="Avatar" class="image">
                         <div class="overlay">HomeTECH #1</div>
                    </div></a>
                    
              </div>
         </section>
         <section class="about-me container" id="aboutme">
              <div class="about-img row">
                   <img src="images/about me.png" alt="" class="col-sm-12 ">
              </div>
              <div class="about-txt row">
                   <h2>about me </h2>
                   <h3>who am i</h3>
                   <p>Passionate about learning and excellence<br>
                     in IT as demonstrates my basic to intermediary<br>
                      knowledge in <strong>Business Intelligence/HTML/CSS/ BOOTSTRAP</strong> gained<br>
                      through the participation in professional training<br>
                     courses and individualy learning. Analytic and<br>
                     practical approach with a great attention to<br>
                     technical details.Reliable and organized, with<br>
                    excellent sales qualifications, as the Business<br>
                     Management Diploma demonstrates. Vast <br>
                     experience in the business field with great sales<br>
                     and negotiation abilities in a wide range of roles<br>
                      and success proven contracts. Flexible, <br>
                      adaptable,and professional, with a great <br>
                      capacity of reaching goals in the set <br>
                      timeframe with an emphasis on details.</p>
              </div>
         </section>
         <section class="item">
              <div class="item-content container">
                   <h3>Goals determine<br>
                     your priorities, and priorities<br>
                      decide if you will achieve your goals.</h3>

              </div>
         </section>
         <section class="profile" id="myprofile">
              <h2>Profile</h2>
              <h4>Jr.Front-End Developer</h4>
              <div class="profile-content container">
                   <div class="profile-txt flex-column row">
                        <div class="text col-sm-12">
                              <h4><strong>About me</strong></h4>
                              
                              <p> I am a junior programmer<br>
                                   with good knowledge of <br>
                                   front-end techniques.<br>
                                   I can acquire the necessary<br> 
                                   knowledge very quickly <br>
                                   and I like to code.<br>
                         </div>
                           
                   </div>
                   <div class="profile-img row">
                        <img src="images/meme.png" alt="" class="imageprf col-sm-12">
                   </div>
                   <div class="profile-txt2 flex-column row">
                        <div class="text col-sm-12">
                              <h4><strong>Details</strong></h4>
                              <br>
                              <h4><strong>Name:</strong></h4>
                              <h5>Serban Adrian</h5>
                              <h4><strong>Age:</strong></h4>
                              <h5>31</h5>
                              <h4><strong>Location:</strong></h4>
                              <h5>Bucharest, Romania</h5>
                        </div>     
                   </div>
              </div>
         </section>
         <footer>
              <div class="footer-content container">
                    <div class="footer-social row">
                         <a href="https://www.facebook.com/serban.adrian.923"><img src="images/facebook.png" alt=""></a>
                         <a href="https://www.instagram.com/adrianserban91/"><img src="images/instagram.png" alt=""></a>
                         <a href="https://www.linkedin.com/in/adrian-serban-89870722a"><img src="images/linkedin.png" alt=""></a>
                    </div>
                    <div class="footer-txt row">
                         <div class="social-txt ">
                              <img src="images/mailk.png" alt="">
                              <h6>adrianserban991@gmail.com</h6>
                         </div>
                         <div class="social-txt ">
                              <img src="images/phone (7).png" alt="">
                              <h6>+40761139888</h6>
                         </div>
                    </div>
               </div>
         </footer>
 
         <script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-fQybjgWLrvvRgtW6bFlB7jaZrFsaBXjsOMm/tB9LTS58ONXgqbR9W8oWht/amnpF" crossorigin="anonymous"></script>
    </body> 
            
    
 
</html>
