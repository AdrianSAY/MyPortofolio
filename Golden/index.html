<!DOCTYPE html>

<html lang="en">

    <head>
         <meta charset="UTF-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <title>Golden</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;700&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap" rel="stylesheet">
         <link rel="stylesheet" href="css/styles.css">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" crossorigin="anonymous">
    </head>
          
    <body>
        
         <section class="hero">
             <div class="container">
                     <div class="header row">
                           <div class="logo col-sm-12 col-md-6">
                                <h1>Golden</h1>
                           </div>
                           <div class="menu  col-sm-12 col-md-6">
                                <ul>
                                     <li a href="#">menu</li>
                                     <li a href="#">service</li>
                                     <li a href="#">portofolio</li>
                                     <li a href="#">about</li>
                                     <li a href="#">contact</li>
                                </ul>
                           </div>
                     </div>
                     <div class="title-content flex-column row">
                          <div class="title col-sm-12 col-md-6 col-lg-12">
                             <h1>Welcome To Our Studio</h1>
                          </div>
                          <div class="subtitle col-sm-12 col-md-12 col-lg-12">
                               <h2>it's nice to meet you </h2>
                          </div>
                          <div class="button-hero col-sm-12 col-md-12 col-lg-12">
                               <button>tell me more</button>
                          </div>

                     </div>
             </div>
         </section>

         <section class="services">
              <div class="services-content container">
                   <div class="services-title flex-column row">
                        <h2>services</h2>
                        <p>Proin iaculis purus consequat sem cure</p>
                   </div>
                   <div class="services-example row">
                        <div class="commerce col-sm-12 col-md-6 col-lg-4">
                               <img src="images/comerce.png" alt="">
                               <h3 class="services-mini-title">E-Commerce</h3>
                               <p class="services-mini-subtitle">Proin iaculis consequat sem cure<br>
                                    digni ssim. Donec porttitora etum suscipit<br>
                                   aenean rhoncus posuere odio in tincidunt.</p>
                        </div>

                        <div class="commerce col-sm-12 col-md-6 col-lg-4 ">
                               <img src="images/responsive.png" alt="">
                               <h3 class="services-mini-title">Responsive Web</h3>
                               <p class="services-mini-subtitle">Proin iaculis consequat sem cure<br>
                                digni ssim. Donec porttitora etum suscipit<br>
                               aenean rhoncus posuere odio in tincidunt.</p>
                         </div>

                         <div class="commerce col-sm-12 col-md-6 col-lg-4">
                               <img src="images/security.png" alt="">
                               <h3 class="services-mini-title">Web Security</h3>
                               <p class="services-mini-subtitle">Proin iaculis consequat sem cure<br>
                                  digni ssim. Donec porttitora etum suscipit<br>
                                 aenean rhoncus posuere odio in tincidunt.</p>
                         </div>
                   </div>

              </div>
                   
              </div>
         </section>

         <section class="our-portofolio ">
            <div class="our-all container">
               <div class="our-portofolio-title flex-column row">
                    <h2>our portofolio</h2>
                    <p>Proin iaculis purus consequat sem cure</p>
               </div>
               <div class="example-portofolio row">
                    <div class="example col-sm-12 col-md-12 col-lg-4">
                              <img src="images/portofolio1.png" alt="">
                              <div class="image-text col-md-12">
                              <h4>Ebony&Ivory</h4>
                              <p>Branding</p>
                              </div>
                    </div>

                    <div class="example col-sm-12 col-md-12 col-lg-4">
                         <img src="images/portofolio2.png" alt="">
                         <div class="image-text col-md-12">
                         <h4>Smart Stationary</h4>
                         <p>Print Design</p>
                         </div>
                    </div>

                    <div class="example col-sm-12 col-md-12 col-lg-4">
                         <img src="images/portofolio3.png" alt="">
                         <div class="image-text col-md-12">
                         <h4>Clever Poster</h4>
                         <p>Print Design</p>
                         </div>
                    </div>
               </div>

               <div class="example-portofolio2 row">
                    <div class="example col-sm-12 col-md-12 col-lg-4">
                              <img src="images/portofolio4.png" alt="">
                              <div class="image-text col-md-12">
                              <h4>Vinyl Record</h4>
                              <p>Product Mock-UP</p>
                              </div>
                    </div>

                    <div class="example col-sm-12 col-md-12 col-lg-4">
                         <img src="images/portofolio5.png" alt="">
                         <div class="image-text col-md-12">
                         <h4>Treehouse template</h4>
                         <p>Web Design</p>
                         </div>
                    </div>

                    <div class="example col-sm-12 col-md-12 col-lg-4">
                         <img src="images/portofolio6.png" alt="">
                         <div class="image-text col-md-12">
                         <h4>Burned Logo</h4>
                         <p>Branding</p>
                         </div>
                    </div>
               </div>
          </div>   
         </section>

         <section class="about-us container">
              <div class="about-us-title flex-column row">
                   <h2>about us</h2>
                   <p>Proin iaculis purus consequat sem cure</p>
              </div>
              <div class="about-us-items">
                    <div class="july-example flex-column-reverse flex-sm-row row">
                         <div class="july-txt col-sm-12 col-md-4 ">
                                   <h5>july 2010</h5>
                                   <h6>Our Humble Beginnings</h6>
                                   <p>Proin iaculis purus consequat sem cure<br>
                                   digini ssim. Donec porttitora entum suscipit<br>
                                   aenean rhoncus posuere odio in tincidunt. Proin<br>
                                   iaculis purus consequat sem cure digni ssim.<br>
                                   Donec porttitora entum suscipit.</p>
                         </div>
                         <div class="july-img col-sm-12 col-md-4">
                                   <img src="images/example1.png" alt="">
                         </div>
                    </div>
               
                              
                    <div class="january-example flex-row row">
                         <div class="january-img col-sm-12 col-md-4">
                                   <img src="images/january.png" alt="">
                         </div>
                         <div class="january-txt col-sm-12 col-md-4 ">
                                   <h5>january 2011</h5>
                                   <h6>Facing Startup Battles</h6>
                                   <p>Proin iaculis purus consequat sem cure<br>
                                   digini ssim. Donec porttitora entum suscipit<br>
                                   aenean rhoncus posuere odio in tincidunt. Proin<br>
                                   iaculis purus consequat sem cure digni ssim.<br>
                                   Donec porttitora entum suscipit.</p>
                         </div>
                    </div> 
                         
                    <div class="december-example flex-row row"> 
                         <div class="december-txt col-sm-12 col-md-4 ">
                                   <h5>December 2012</h5>
                                   <h6>Enter The Dark Days</h6>
                                   <p>Proin iaculis purus consequat sem cure<br>
                                   digini ssim. Donec porttitora entum suscipit<br>
                                   aenean rhoncus posuere odio in tincidunt. Proin<br>
                                   iaculis purus consequat sem cure digni ssim.<br>
                                   Donec porttitora entum suscipit.</p>
                         </div>
                         
                         <div class="december-img col-sm-12 col-md-4">
                              <img src="images/december.png" alt="">
                         </div>
                    </div>    
               
                              
                    <div class="february-example flex-row row">
                         <div class="february-img col-sm-12 col-md-4">
                              <img src="images/february.png" alt="">
                         </div>
                         <div class="february-txt col-sm-12 col-md-4 ">
                              <h5>february 2014</h5>
                              <h6>Our Triumph</h6>
                              <p>Proin iaculis purus consequat sem cure<br>
                              digini ssim. Donec porttitora entum suscipit<br>
                              aenean rhoncus posuere odio in tincidunt. Proin<br>
                              iaculis purus consequat sem cure digni ssim.<br>
                              Donec porttitora entum suscipit.</p>
                         </div>
                    </div>

                    <div class="about-example flex-row row">             
                         <div class="our-story-example col-sm-12 col-lg-12">
                              <div class="our-story-img">
                                   <img src="images/our story.png" alt="">
                              </div>
                              
                         </div> 
                    </div>
               </div>        
         </section>
      
          <section class="team">
               <div class="team-content container">
                     <div class="team-title flex-column row">
                         <h3>Our amazing team</h3>
                         <p>Proin iaculis purus consequat sem cure</p>
                     </div>
               </div>      

               <div class="allteam row">    
                     <div class="team-pers col-sm-12 col-md-4">
                          <div class="blabla ">
                         <img src="images/team.png" alt="">
                         <h5>Kimberly Thompson</h5>
                         <p>Marketer</p>
                      <div class="images-team">  
                         <div class="social">
                              <img src="images/Twitter.png" alt="">
                         </div>
                         <div class="social">
                              <img src="images/Facebook.png" alt="">
                         </div>
                         <div class="social">
                              <img src="images/Pinterest.png" alt="">
                         </div>
                         </div>
                         </div>
                    </div>

                    <div class="team-pers col-sm-12 col-md-4">
                         <div class="blabla ">
                         <img src="images/team.png" alt="">
                         <h5>Rico Masimo</h5>
                         <p>Coder</p>
                    <div class="images-team">  
                         <div class="social">
                              <img src="images/Twitter.png" alt="">
                         </div>
                         <div class="social">
                              <img src="images/Facebook.png" alt="">
                         </div>
                         <div class="social">
                              <img src="images/Pinterest.png" alt="">
                         </div>
                         </div>
                         </div>
                    </div>        
                    

                    <div class="team-pers col-sm-12 col-md-4">
                         <div class="blabla">
                              <img src="images/team.png" alt="">
                              <h5>Uku Mason</h5>
                              <p>Graphic Designer</p>
                         <div class="images-team">  
                         <div class="social">
                              <img src="images/Facebook.png" alt="">
                         </div>
                         <div class="social">
                              <img src="images/Pinterest.png" alt="">
                         </div>
                         </div>
                         </div>  
                   
                    </div>     
               </div> 

               <div class="text">
                    <p>Proin iaculis purus consequat sem cure digni ssim dcnec portitora entum suscipit<br>
                        aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
               </div>
          </section>
          <section class="partener container">
          
               <div class="parteners row">
                    <img class="imo col-sm-12 col-md-3" src="images/p1.png" alt="">            
                    <img class="imo col-sm-12 col-md-3" src="images/p2.png" alt="">
                    <img class="imo col-sm-12 col-md-3" src="images/p3.png" alt="">
                    <img class="imo col-sm-12 col-md-3" src="images/p4.png" alt="">
               </div>
          
          </section>
          <section class="contactus">
            <div class="contact container">
               <div class="title flex-column row">
                    <h3>contact us</h3>
                    <p>Proin iaculis purus consequat sem cure</p>
               </div>
               <div class="tanana  row">
               <div class="mesaje-contain col-sm-12 col-lg-6">
                    <div class="your-name">
                         <input type="text" id="your" name="fname" placeholder="YOUR NAME*">
                    </div>
                    <div class="your-email">
                        <input type="text" id="your" name="fname" placeholder="YOUR E-MAIL*">
                    </div>
                          
                    <div class="subject">
                         <input type="text" id="your" name="fname" placeholder="SUBJECT*">
                    </div>

               </div>   
               <div class="bruuu col-sm-12 col-lg-6">
                    <form action="/action_page.php">
                         <textarea id="huh" name="huh" rows="5" cols="40">
                          YOUR MESAGE*
                           </textarea>
                    </form>
                </div>  
               </div>
               <div class="button">
                    <button>SEND MESSAGE</button>
               </div>
            </div>
          </section>
          <section class="footer container">
                 <div class="copyright row">
                    <div class="copyright-txt col-lg-3 ">
                      <p class="cpy">©Copyright 2014 FreebiesXpress.com</p>
                    </div>
                    <div class="copyright-img col-lg-3 ">
                         
                         <img src="images/f1.png" alt="">
                         <img src="images/f2.png" alt="">
                         <img src="images/f3.png" alt="">
                         <img src="images/f4.png" alt="">
                    

                    </div>

                 </div>
          </section>



         <script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-fQybjgWLrvvRgtW6bFlB7jaZrFsaBXjsOMm/tB9LTS58ONXgqbR9W8oWht/amnpF" crossorigin="anonymous"></script>
    </body> 
            
    
 
</html>